stock_barcodes_picking_batch/wizard/stock_barcodes_read_picking_batch.py

```diff
@@ -0,0 +1,357 @@
+# Copyright 2019 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import logging
+
+from odoo import _, api, fields, models
+from odoo.tools.float_utils import float_compare
+
+_logger = logging.getLogger(__name__)
+
+
+class WizStockBarcodesReadPickingBatch(models.TransientModel):
+    _inherit = "wiz.stock.barcodes.read.picking"
+    _description = "Wizard to read barcode on picking batch"
+
+    picking_batch_id = fields.Many2one(
+        comodel_name="stock.picking.batch", string="Picking Batch", readonly=True
+    )
+    candidate_picking_batch_ids = fields.One2many(
+        comodel_name="wiz.candidate.picking.batch",
+        inverse_name="wiz_barcode_id",
+        string="Candidate picking batch",
+        readonly=True,
+    )
+    # TODO: Remove this field
+    picking_batch_product_qty = fields.Float(
+        string="Picking batch quantities",
+        digits="Product Unit of Measure",
+        readonly=True,
+    )
+    picking_type_code = fields.Selection(
+        [("incoming", "Vendors"), ("outgoing", "Customers"), ("internal", "Internal")],
+        "Type of Operation",
+    )
+    confirmed_moves = fields.Boolean(string="Confirmed moves")
+    picking_mode = fields.Selection(
+        selection_add=[("picking_batch", "Picking batch mode")]
```

Comment:
No `ondelete` here?

Reply:
Is needed for transient models that only run once time?

Reply:
I have added anyway the ondelete attribute